### DISCUSSION TITLE
model mesh

### DISCUSSION THREAD

**Discussion by anirbanroy16** [pybamm-team/PyBaMM#3450] [Q&A]
What do R_n and R_p mean? number of current collector mesh points, by any chance? I get it r_n and r_p refer to mesh points (or # of onion "slices") in the particle.

![image](https://github.com/pybamm-team/PyBaMM/assets/25733848/3d5c9dcd-26fa-480b-8883-57d0996590d3)

**Comment by brosaplanella** [comment]
`R_n` and `R_p` refer to the particle size variable for models with particle size distributions (e.g. https://docs.pybamm.org/en/latest/source/examples/notebooks/models/MPM.html).
